platform/security/idp/security-idp-client/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -17,9 +17,9 @@
 
     <OCD name="Logout Page" id="org.codice.ddf.security.idp.client.LogoutRequestService">
         <AD
-                name="Log Out Page Time Out" id="logOutPageTimeOut" required="true" type="Long"
+                name="Logout Page Time Out" id="logOutPageTimeOut" required="true" type="Long"
                 default="3600000"
-                description="This is the time limit that the IDP client will wait for a user to click log out on the log out page. Any requests that take longer than this time for the user to submit will be rejected."/>
+                description="This is the time limit that the IDP client will wait for a user to click log out on the logout page. Any requests that take longer than this time for the user to submit will be rejected."/>
```

Comment:
I guess this doesn't have the unit either. It'd probably make it much clearer

Reply:
fixed